.github/workflows/ci.yaml

```diff
@@ -34,42 +34,51 @@ jobs:
       - name: Install required Python packages
         run: |
           python -m pip install --upgrade pip
-          python -m pip install nox wheel
+          python -m pip install nox
 
       - name: run nox
         run: nox -e ci_checks_max pytest_min
         timeout-minutes: 10
 
-  build-wheels:
+  build-dist:
     runs-on: ubuntu-20.04
-    needs: test
-
     steps:
-    - name: Fetch sources
-      uses: actions/checkout@v3
+      - name: Fetch sources
+        uses: actions/checkout@v3
 
-    - name: Set up Python
-      uses: actions/setup-python@v4
-      with:
-        python-version: 3.8
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

Comment:
Why change `3.8` to `3.10`? Is it the python version for which this job will publish `sdk` to PyPi?